content/swagger/resources/asset_categories/routes/asset_categories.yaml

```diff
@@ -0,0 +1,169 @@
+                  _items:
+                    type: array
+                    items:
+                      $ref: '#/definitions/AssetCategoryList'
+      examples: {
+        _links: {
+          self: {
+            href: "https://demo.akeneo.com/api/rest/v1/asset-categories?page=2&limit=3"
+          },
+          first: {
+            href: "https://demo.akeneo.com/api/rest/v1/asset-categories?page=1&limit=3"
+          },
+          previous: {
+            href: "https://demo.akeneo.com/api/rest/v1/asset-categories?page=1&limit=3"
+          },
+          next: {
+            href: "https://demo.akeneo.com/api/rest/v1/asset-categories?page=3&limit=3"
+          }
+        },
+        current_page: 2,
+        _embedded: {
+          _items: [
+            {
+              "_links": {
+                "self": {
+                  "href": "https://demo.akeneo.com/api/rest/v1/asset-categories/pictures"
+                }
+              },
+              "code": "pictures",
+              "parent": null,
+              "labels": {
+                "en_US": "Pictures",
+                "fr_FR": "Images",
+                "de_DE": "Bilder"
+              }
+            },
+            {
+              "_links": {
+                "self": {
+                  "href": "https://demo.akeneo.com/api/rest/v1/asset-categories/front_views"
+                }
+              },
+              "code": "front_views",
+              "parent": "pictures",
+              "labels": {
+                "en_US": "Front views",
+                "fr_FR": "Vues de face",
+                "de_DE": "Vorderansichten"
+              }
+            },
+            {
+              "_links": {
+                "self": {
+                  "href": "https://demo.akeneo.com/api/rest/v1/asset-categories/side_views"
+                }
+              },
+              "code": "side_views",
+              "parent": "pictures",
+              "labels": {
+                "en_US": "Side views",
+                "fr_FR": "Vues de profil",
+                "de_DE": "Profilansichten"
+              }
+            }
+          ]
+        }
+      }
+    401:
+      $ref: "#/responses/401Error"
+    403:
+      $ref: "#/responses/403Error"
+    406:
+      $ref: "#/responses/406Error"
+post:
+  summary: Create a new asset category
+  operationId: "post_asset_categories"
+  description: This endpoint allows you to create a new asset category.
+  tags:
+    - Asset categories
+  x-versions:
+    - 2.1
+  x-ee: true
+  parameters:
+    - name: body
+      in: body
+      schema:
+        $ref: "#/definitions/AssetCategory"
+  responses:
+    201:
+      $ref: "#/responses/Created"
+    400:
+      $ref: "#/responses/400Error"
+    401:
+      $ref: "#/responses/401Error"
+    403:
+      $ref: "#/responses/403Error"
+    415:
+      $ref: "#/responses/415Error"
+    422:
+      $ref: "#/responses/422Error"
+patch:
+  summary: Update/create several asset categories
```

Comment:
Updates/creates ?

Reply:
It's the endpoint name. Not a real verb here.